tests/test_zdd.py

```diff
@@ -329,11 +423,82 @@ def test_hybrid(self):
 
     @mock.patch('requests.get',
                 mock.Mock(side_effect=lambda k, auth:
+                          MyResponse('tests/zdd_app_blue_marathon1.5.json')))
+    def test_hybrid_marathon15(self):
```

Comment:
So this is just the marathon15 version of `test_hybrid` which seems to be a slight modification of `test_simple`. The only difference I see is that `hybrid` increases the `HAPROXY_DEPLOYMENT_NEW_INSTANCES` to 1.

I have no idea where the `hybrid` name came from, but was attempting to refrain from completely refactoring.